docs/source/requirements.txt

```diff
@@ -9,3 +9,4 @@ markdown
 pandas
 matplotlib
 future
+typing
```

Comment:
Do we need `typing` if using Python 3.6 and above?

Reply:
Good point - removed.